public/.gitignore

```diff
@@ -10,4 +10,7 @@ example.gitignore
 index.php
 robots.txt
 update.php
-web.config
\ No newline at end of file
+web.config
+/autoload.php
+/autoload.php
+/autoload.php
```

Comment:
pitäiskö näitä olla vaan yksi

Reply:
Jaa-a. Mistähän ovat tulleet. 
```

/autoload.php
/autoload.php
/autoload.php
/autoload.php
```

Nyt näyttäs olevan 4. Enkä oo yhtään lisänny ite :)

Joku skripti solmussa taas.